doc/quickstart/pycsw_quickstart.rst

```diff
@@ -15,71 +16,66 @@ pycsw is an OGC CSW server implementation written in `Python <https://python.org
   :scale: 75 %
 
-By selecting "GetCapabilities-SOAP" and pushing the "Send" button, a SOAP request is sent to the server to advertise their web service capabilities:
+3. Selecting "GetCapabilities-SOAP" and press the "Send" button. A SOAP request is sent to the server to advertise their web service capabilities.
 
 .. image:: /images/projects/pycsw/pycsw_tester_soap_capabillities.png
   :scale: 75 %
 
-Also, the user can search for data records, performing a spatial bounding box query, by selecting "GetRecords-filter-bbox" and editing the coordinates in the XML request:
+4. You can search for data records, performing a spatial bounding box query, by selecting "GetRecords-filter-bbox" and editing the coordinates in the XML request.
 
 .. image:: /images/projects/pycsw/pycsw_tester_getrecords_bbox_filter.png
   :scale: 75 %
 
-The user can go through all the available requests and perform various requests from this testing application.
+You can go through all the available requests and perform various requests from this testing application.
 
-Capabilities Document and Configuration
+Understand capabilities and configuration
 =======================================
 
-The capabilities of the pycsw installation can be found at ``http://localhost/pycsw?service=CSW&version=2.0.2&request=GetCapabilities``. If you just use ``http://localhost/pycsw`` you will get the capabilities document for CSW 3.0.0, since that is the latest version supported.
+The capabilities of the pycsw installation can be found at ``http://localhost/pycsw?service=CSW&version=2.0.2&request=GetCapabilities``. 
+If you just use ``http://localhost/pycsw`` you will get the capabilities document for CSW 3.0.0, since that is the latest version supported.
 
-To edit the web service metadata, included in the capabilities document, the user can modify the file ``/etc/pycsw/default.cfg`` under the tag ``[metadata:main]``.
+To edit the web service metadata, included in the capabilities document, you can modify the file ``/etc/pycsw/default.cfg`` under the tag ``[metadata:main]``.
 
-If the user wants to enable INSPIRE Discovery Service support, the metadata under the tag ``[metadata:inspire]`` has to be filled in and the ``enabled`` property has to be set to ``true``.
+If you want to enable INSPIRE Discovery Service support, fill in the metadata under the tag ``[metadata:inspire]`` and set the ``enabled`` property to ``true``.
 
 For full functionality and configuration options, please consult the `documentation`_ on the pycsw website.
 
-Metadata Creation
+Create metadata
 =================
 
 Metadata files for a spatial dataset or a spatial web service can be created through the open source implementation of inspire directive for metadata, European Open Source Metadata Editor (EUOSME). This application can be found at ``http://inspire-geoportal.ec.europa.eu/editor/``. Source code is available at ``https://joinup.ec.europa.eu/svn/euosme/trunk``
```

Comment:
I think the metadata page has moved. `http://inspire-geoportal.ec.europa.eu/editor/` is now possibly this instead: http://inspire.ec.europa.eu/validator/